src/pystatis/http_helper.py

```diff
@@ -99,22 +100,7 @@ def get_data_from_endpoint(endpoint: str, method: str, params: dict, db_name: st
         name = params.get("name", params.get("selection", ""))
 
         if name is not None:
-            db_match = db.identify_db(name)
-
-            # Check credentials (Note: we might want to do this also for explicitly specified db_names?)
-            # If more than one db matches it must be a Cube (provided all regexing works as intended).
-            # --> Choose db based on available credentials.
-            if db_match:
-                for name in db_match:
-                    if db.check_credentials(name):
-                        db_name = name
-                        break
-                else:
-                    raise PystatisConfigError(
-                        "Missing credentials!\n"
-                        f"To access this item you need to be a registered user of: {db_match} \n"
-                        "Please run setup_credentials()."
-                    )
+            db_name = identify_db(name)
 
     if not db_name:
```

Comment:
If I am not mistaken, your changes to `identify_db` make this case impossible as `db_name` always exists at this point as otherwise there would have been an exception. So we either remove this check here or we use a try except blocvk around identify_db and in case of any exception we set `db_name=None` or empty string...but does not make much sense to catch an exception just to throw it again...so I think we can safely remove it from here, correct?

Reply:
Good catch!